recipe/yum_requirements.txt

```diff
@@ -1,6 +1,6 @@
 csh
 imake
-libx11-devel
+libX11-devel
```

Comment:
@khallock changes in the `yum_requirements.txt` file need a `rerender` to propagate them. Are you familiar with conda-smithy and re-rendering?